usr/share/rear/prep/GNU/Linux/31_include_cap_uitls.sh

```diff
@@ -0,0 +1,5 @@
+# include utilities needed to set capabilities
+if [ "$BACKUP_CAP" == "y" ] ; then
+    REQUIRED_PROGS=("${REQUIRED_PROGS[@]}" setcap getcap)
+    Log "Tools to set capabilities."
```

Comment:
If you need to have a log message, maybe call it "Include tools to save/restore capabilites"